src/app/landing/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Image } from '@chakra-ui/image';
+import { Flex, Grid, GridItem, Heading, Text } from '@chakra-ui/layout';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+export default function Landing() {
+  const [isLargerThan1100] = useMediaQuery('(min-width: 1100px)');
+  console.log(isLargerThan1100);
+
+  return (
+    <Grid
+      templateColumns={isLargerThan1100 ? 'repeat(2, 1fr)' : 'repeat(1, 1fr)'}
+      px="150px"
+      gap="81px"
+      justifyItems="center"
+      w="100%"
+      h="100%"
+      alignItems="center"
+    >
+      <GridItem w={{ lg: '400px', xl: '639px', xxl: '1000px' }}>
+        <Heading fontSize="75.68px" lineHeight="100px" mb="28px">
+          Explore UVic Courses
+        </Heading>
+        <Text fontSize="25.71">CourseUp makes it simple to browse and schedule UVic Courses</Text>
+        <Flex alignItems="center" mt="40px" as="a" w="fit-content" target="_blank" href="https://www.vikelabs.ca">
+          <Text color="#4C6EA5" fontSize="24.25" mr="5px">
+            Built by students @
+          </Text>
+          <Image src={process.env.PUBLIC_URL + '/assets/vikelabs.svg'} alt="VikeLabs" />
+        </Flex>
+      </GridItem>
+      {isLargerThan1100 && (
+        <GridItem maxH="563px" minW="fit-content">
```

Comment:
You could do `display: none`?

Reply:
wdym?

Reply:
I get `display: none` but curious why

Reply:
My perceived usage of `useBreakpoint` was moreso for things JS like maybe changing the behaviour of how a call is made or something and not for stuff like this is my take on it. Also the `sm, xl`, etc. breakpoints are consistent given they are globally set on the theme.

Reply:
Performance wise I think it's a bit faster since I _think_ there's only an initial runtime cost but with this there is a JS cost each time (not significant for this but speaking generally)

Reply:
I might be missing your idea on the implementation, but I don't think this is working 😕

Reply:
https://chakra-ui.com/docs/features/responsive-styles#the-object-syntax with the keyword being `upwards`.

Reply:
`upwards` 😡